libs/sql-parser/src/main/antlr/io/crate/sql/parser/antlr/SqlBaseParser.g4

```diff
@@ -339,7 +339,8 @@ explicitFunction
     | name=CURRENT_TIMESTAMP
         (OPEN_ROUND_BRACKET precision=integerLiteral CLOSE_ROUND_BRACKET)?           #specialDateTimeFunction
     | CURRENT_SCHEMA                                                                 #currentSchema
-    | (CURRENT_USER | USER)                                                          #currentUser
+    | (CURRENT_USER | CURRENT_ROLE | USER)                                           #currentUser
+    | (CURRENT_USER | CURRENT_ROLE | USER)                                           #currentUser
```

Comment:
why is this line added twice?
We can also use `#currentRole` already here, but also find to keep it as is, your call.

Reply:
Oh, duplicated the line by mistake, thx!